pkg/sql/opt/norm/testdata/rules/join

```diff
@@ -2823,3 +2823,73 @@ full-join (cross)
  │    └── fd: (3)-->(4)
  └── filters
       └── (substring('', ')') = '') = (u:3 > 0) [outer=(3)]
+
+exec-ddl
+CREATE TABLE ab (
+  a INT,
+  b INT,
+  PRIMARY KEY (a,b)
+)
+----
+
+
+exec-ddl
+CREATE TABLE cd (
+  c INT,
+  d INT
+)
+----
+
+exec-ddl
+ALTER TABLE cd INJECT STATISTICS '[
+    {
+        "columns": [ "c" ],
+        "created_at": "2019-12-13 16:17:43.033228+00:00",
+        "distinct_count": 4,
+        "row_count": 4
+    }
+]'
+----
+
+norm
+SELECT a,b FROM ab, cd WHERE a=c AND b=d
+----
+project
+ ├── columns: a:1(int!null) b:2(int!null)
+ └── inner-join (lookup ab)
+      ├── columns: a:1(int!null) b:2(int!null) c:3(int!null) d:4(int!null)
+      ├── key columns: [3 4] = [1 2]
+      ├── lookup columns are key
+      ├── fd: (1)==(3), (3)==(1), (2)==(4), (4)==(2)
+      ├── scan cd
+      │    └── columns: c:3(int) d:4(int)
+      └── filters (true)
+
+
+norm
+SELECT * FROM ab WHERE (a, b) IN (SELECT c, d FROM cd)
```

Comment:
This isn't working is it? We should have removed the `(a,b)` projections from the inputs

Reply:
I'm not sure exactly what you mean here - I've rewritten the testing here so it's more in line with the original issue